### PR TITLE
data: ChinmayMhatre

### PR DESCRIPTION
**Pull request by ChinmayMhatre**
Added an event to my profile.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3886"></a>

### CODE REVIEW

**Comment by ChinmayMhatre** [comment]
Could someone check if the timestamps are correct, I am not sure about them. I want them to be Fri, Feb 3, 3:00 PM - Feb 5, 6:00 PM GMT+5:30

File: data/ChinmayMhatre/events/2023-02-03-hack-this-fall-hackthon.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "isVirtual": false,
+    "isInPerson": true,
+    "color": "red",
+    "name": "Hack This Fall 3.0",
+    "description": "A 36-hour in-person hackathon taking place at Karnavati University in Gandhinagar, Gujarat from February 3rd to 5th, 2023.",
+    "date": {
+      "start": "2023-02-03T16:00:00.000-05:00",
```
Comment (SaraJaoude) [inline data/ChinmayMhatre/events/2023-02-03-hack-this-fall-hackthon.json:8]: I think it should be:

```suggestion
      "start": "2023-02-03T15:00:00.000+05:30",
```

File: data/ChinmayMhatre/events/2023-02-03-hack-this-fall-hackthon.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+      "end": "2023-02-05T17:00:00.000-05:00"
```
Comment (SaraJaoude) [inline data/ChinmayMhatre/events/2023-02-03-hack-this-fall-hackthon.json:9]: And this one:

```suggestion
      "end": "2023-02-05T18:00:00.000+05:30"
```

**Comment by ChinmayMhatre** [comment]
Thanks Sara, Updated the dates.

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**